tests/ui/unnecessary_cast.stderr

```diff
@@ -2,21 +2,21 @@ error: casting to the same type is unnecessary (`i32` -> `i32`)
   --> $DIR/unnecessary_cast.rs:6:5
    |
 LL |     1i32 as i32;
-   |     ^^^^^^^^^^^
+   |     ^^^^^^^^^^^ help: try: `1i32`
```

Comment:
Wouldn't we usually suggest `1_i32` for better legibility?

Reply:
See #6232